sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -73,18 +73,19 @@ example `http://localhost:8080/login/oauth2/code/`. Note the tailing `/` cannot
     
      (B). You can provide one by extending `AADWebSecurityConfigurerAdapter` and call `super.configure(http)` explicitly 
     in the `configure(HttpSecurity http)` function. Here is an example:
-    <!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/aad/AADOAuth2LoginConfigSample.java#L18-L29 -->
+    <!-- embedme ../azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/java/com/azure/spring/sample/aad/security/AADOAuth2LoginSecurityConfig.java#L11-L23 -->
     ```java
     @EnableWebSecurity
     @EnableGlobalMethodSecurity(prePostEnabled = true)
-    public class AADOAuth2LoginConfigSample extends AADWebSecurityConfigurerAdapter {
+    public class AADOAuth2LoginSecurityConfig extends AADWebSecurityConfigurerAdapter {
     
+        /**
+         * Add configuration logic as needed.
+         */
         @Override
         protected void configure(HttpSecurity http) throws Exception {
             super.configure(http);
-            http.authorizeRequests()
-                .antMatchers("/login").permitAll()
-                .anyRequest().authenticated();
+            // Do some custom configuration
```

Comment:
Will all requests getting through without being authenticated?

Reply:
Yes, In web app sample code, the request is not required to be authenticated.

Reply:
@ZhuXiaoBing-cn , Please add `.anyRequest().authenticated();` in the sample project.

Reply:
@chenrujun  Okay.